.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,15 @@
-<!-- Hi, and thank you for submitting a Pull Request! The checklist below is a brief summary of steps found in the NeuroPoly Contributing Guidelines, which can be found here: https://intranet.neuro.polymtl.ca/onboarding/software-development.html#software-development. 
--->
-
 ## Checklist
 
-#### GitHub
-
-- [ ] I've given this PR a concise, self-descriptive, and meaningful title
-- [ ] I've linked relevant issues in the PR body
-- [ ] I've applied [the relevant labels](https://intranet.neuro.polymtl.ca/onboarding/software-development.html#pr-labels) to this PR
```

Comment:
With this link gone, I didn't see a small but important distinction on the wiki page: we want **exactly one** dark purple label, and **zero or more** light purple labels. (This is an assumption made by [the changelog-generation code](https://github.com/neuropoly/changelog/blob/575c8c6105819ae3f42265d083585e77f4248584/changelog/changelog.py#L405-L414)).

Reply:
I've tried to make this extra clear here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4912/commits/8e43a19c178d87afb38844e4512985ca90e1b2ab